draft-ietf-quic-transport.md

```diff
@@ -908,10 +908,13 @@ final size is the number of bytes sent.  More generally, this is one higher
 than the offset of the byte with the largest offset sent on the stream, or zero
 if no bytes were sent.
 
-For a stream that is reset, the final size is carried explicitly in a
-RESET_STREAM frame.  Otherwise, the final size is the offset plus the length of
-a STREAM frame marked with a FIN flag, or 0 in the case of incoming
-unidirectional streams.
+The final size of a stream is always signaled to the recipient.  The final size
+is the sum of the Offset and Length fields of a STREAM frame with a FIN flag,
+noting that the length might be implicit.  Alternatively, the Final Size field
```

Comment:
Maybe "noting that the offset and the length might be implicit"?

This is an editorial nitpick purely from the viewpoint of consistency, but it is a bit surprising to see only one of the two fields called out as might be implicit, even though both of the fields can be omitted.

Reply:
True.  That will only be the case when a stream has only one STREAM frame, so it would be _unusual_ for the final STREAM frame to have an implicit Offset, but it's certainly not prohibited.